lib/ui/painting/codec.cc

```diff
@@ -350,7 +350,8 @@ IMPLEMENT_WRAPPERTYPEINFO(ui, Codec);
   V(Codec, getNextFrame)    \
   V(Codec, frameCount)      \
   V(Codec, repetitionCount) \
-  V(Codec, dispose)
+  V(Codec, dispose)         \
+  V(Codec, clearAndDisableFrameCache)
```

Comment:
Add this in painting.dart as well

Reply:
So there's already a native method in `painting.dart` on line 1583. Should it be added anywhere else too?